### PR TITLE
Add --recursive option to doc sync command

### PR DESCRIPTION
**Pull request by ollyfg**
This option causes a simple recursive search of the supplied folder, nothing fancy.

### CODE REVIEW

**Comment by erunion** [comment]
@ollyfg Can you rebase the latest `main` into this branch? We've moved some things around in the process of dropping support for Node 10.

**Comment by erunion** [comment]
@rahulhegdee This might collide with the work that you're planning on doing, you mind reviewing this? ty

**Review by rahulhegdee — CHANGES_REQUESTED** [review]
Small fix but otherwise, looks good to me!

File: src/cmds/docs/index.js [hunk 1]
```diff
@@ -35,6 +40,7 @@ exports.args = [
 ];
 
 exports.run = function (opts) {
+  const { folder, key, version, recursive } = opts;
   const { folder, key, version } = opts;
```
[the remark this answers is not in the document]
Comment (ollyfg) [reply src/cmds/docs/index.js]: Whoops! Sorry, must have snuck in there while rebasing.

**Review by rahulhegdee — APPROVED** [review]
👍

**Review by erunion — APPROVED** [review]

**Review by erunion — APPROVED** [review]

**Review by rahulhegdee — APPROVED** [review]

**Pull request merged**